adapters/triplelift/triplelift_test.go

```diff
@@ -6,5 +6,5 @@ import (
 )
 
 func TestJsonSamples(t *testing.T) {
-	adapterstest.RunJSONBidderTest(t, "triplelifttest", NewTripleliftBidder(nil, "http://tlx.3lift.net/s2s/auction?supplier_id=19"))
+	adapterstest.RunJSONBidderTest(t, "triplelifttest", NewTripleliftBidder(nil, "http://tlx.3lift.net/s2s/auction?supplier_id=20"))
```

Comment:
The build failed for this PR because there are tests defined in the json files of `triplelifttest/exemplary` and `triplelifttest/supplemental` which still reference the id of 19.

Reply:
Should be resolved now

Reply:
Yup. Looks good now.